data/comid/README.md

```diff
@@ -0,0 +1,83 @@
+-   **tag-identity** (`Object`): Uniquely identifies this CoMID document via an ID (often a UUID) and includes a version number.
+-   **entities** (`Array`): Lists the entities (organizations, individuals, etc.) contributing to or maintaining the document, along with their roles.
+
+### 2.2 Triples
+
+-   **reference-values**: One or more **reference-value** objects, each containing an **environment** and one or more **measurements**.
+-   **attester-verification-keys**: One or more **attester-verification-key** objects, each containing an **environment** and an array of **verification-keys**.
+
+
+## 3 Components
+
+### 3.1 Environment
+
+An **environment** captures the context of a measurement or verification key:
+
+-   **class**: Vendor, model, and possibly an ID (`type` + `value`).
+-   **instance** (`optional`): For distinguishing multiple instances of the same environment (e.g., using `ueid` or `uuid`).
+-   **layer** and **index** (`optional`): For layered environments (e.g., DICE layers in multi-stage boot processes).
+
+### 3.2 Measurements
+
+Each measurement has two crucial subfields:
+
+-   **key**: Identifies the measurement, including possible fields like `label`, `version`, and `signer-id`.
+-   **value**: Holds the actual measurement data (e.g., cryptographic digests, raw values, or operational flags).
+
+### 3.3 Attester Verification Keys
+
+Used to store **public keys** associated with an environment. This is essential for verifying the attestation claims or measurement signatures produced by that environment.
+
+## 4 Field-By-Field Explanation
+
+### 4.1 Global Fields
+|     Field    	|  Type  	|                       Description                       	|                    Example                   	|   	
+|:------------:	|:------:	|:-------------------------------------------------------:	|:--------------------------------------------:	|
+| lang         	| String 	| Language/country code.                                  	| "en-GB"                                      	|   	
+| tag-identity 	| Object 	| Identity of this CoMID tag (UUID + version).            	| "id": "43BBE37F-2E61-4B33-AED3-53CFF1428B16" 	|   	
+| entities     	| Array  	| The organizations/roles associated with this CoMID tag. 	| [ { "name": "ACME Ltd." ... } ]              	|   	
+
+### 4.2 Reference-Value Fields
+|        Field       |  Type  |                                     Description                                    |                                            Example                                            |
+|:------------------:|:------:|:----------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------:|
+| environment        | Object | Contains class and optionally instance, layer, index.                              | See 3.1 Environment.                                                                          |
```

Comment:
This enumeration confuses what is optional and required, as well as what is where. The layer and index are optional fields of class, whereas instance is at the same level as class.

Reply:
@ravjot07 take a look at these here:- https://github.com/ravjot07/cocli/tree/docs/data/comid#42-reference-value-fields 
actually i had simply used excel file for these tables and then later converted it into md format using online tools like table converter